eth/backend.go

```diff
@@ -151,11 +151,14 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 			log.Error("Failed to recover state", "error", err)
 		}
 	}
+
 	// Transfer mining-related config to the ethash config.
 	chainConfig, err := core.LoadChainConfig(chainDb, config.Genesis)
 	if err != nil {
 		return nil, err
 	}
+	// Hardcode TerminalTotalDifficultyPassed to true.
+	chainConfig.TerminalTotalDifficultyPassed = true
```

Comment:
Yes, again it's the same change. Since this should be set in the migration script we will be removing this code and as such I didn't comment it heavily.